docs/manual/setup/getting-started.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Getting Started with Vector
+description: Getting Started with Vector, the "Hello World" of Vector tutorials.
+---
+
+Welcome to Vector! 🎈
+
+Vector is a high-performance observability data platform that allows you to collect, transform, and route all your
+logs and metrics.
+
+In this getting started guide, we'll walk you through using Vector for the first time. We'll install Vector
+and create our first observability data pipeline so you can begin to see what Vector can do.
+
+## Install Vector
+
+Installing Vector is quick and easy. We can use this handy installation script:
+
+```shell
+curl --proto '=https' --tlsv1.2 -sSf https://sh.vector.dev | sh
+```
```

Comment:
I'd like to show the output of these commands, we can do that in a follow up though.

Reply:
The reason I didn't is the same as using `generator` - it'll look different on different platforms. Prior experience of "my screen doesn't look like that" has guided me on that. :)